lib/search/models/search_sort_by.dart

```diff
@@ -1,6 +1,10 @@
 enum SearchSortBy {
-  relevance,
-  rating,
-  upload_date,
-  view_count;
+  relevance("Relevance"),
```

Comment:
We need to use the locals here. Which is not super convenient. What i usually do is create a function within the enum that takes in the localisations as a param. See example here: https://github.com/lamarios/clipious/blob/master/lib/home/models/db/home_layout.dart#L45